lib/modules/manager/bundler/__snapshots__/extract.spec.ts.snap

```diff
@@ -542,12 +542,15 @@ exports[`modules/manager/bundler/extract extractPackageFile() parse mastodon Gem
       },
     },
     {
+      "currentDigest": "54b17ba8c7d8d20a16dfc65d1775241833219cf2",
       "currentValue": "'~> 0.6'",
-      "datasource": "rubygems",
```

Comment:
Why this change? We were missing this one because of an un-handled line order. This refactor allow finer extraction, and allow for such fixes.